lib/views/walks/walks_view.dart

```diff
@@ -289,7 +286,7 @@ class _WalksViewState extends State<WalksView> with WidgetsBindingObserver {
           }
         }
 
-        if (mounted && toShow.isNotEmpty) {
+        if (toShow.isNotEmpty) {
```

Comment:
Not sure why you removed the check on "mounted", for me that's an important check to ensure that we are not trying to display the dialog on a now-dismissed screen (e.g. user taped directly the settings tabs before the news were received)

Reply:
I don't really see it as a problem if news screen is shown outside of the 'calendar' screen...

Reply:
No that's not really what I mean, I think that the dialog is linked to the "calendar" screen because that's where it is displayed, which means that if the screen is gone, then it will not be displayed properly and might throw an error because of that

Reply:
True, I'll test it. It's because I have the 'opt in to crashlytics' dialog in mind, which is similar to this, where the code starts in initscreen, but the screen is only shown in the 'calendar' screen. But there the initScreen scaffold is still in the widget tree, so it's different. I'll double check.